aspnetcore/fundamentals/app-state.md

```diff
@@ -1,236 +1,191 @@
 ---
-title: Managing Application State | Microsoft Docs
+title: Session and application state | Microsoft Docs
 author: rick-anderson
-description: 
-keywords: ASP.NET Core,
+description: Approaches to preserving application and user (session) state between requests.
+keywords: ASP.NET Core, Application state, session state, querystring, post
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 1/14/2017
 ms.topic: article
 ms.assetid: 18cda488-0769-4cb9-82f6-4c6685f2045d
 ms.technology: aspnet
 ms.prod: aspnet-core
 uid: fundamentals/app-state
 ---
-# Managing Application State
+<!-- Review questions
 
->[!WARNING]
-> This page documents version 1.0.0-rc1 and has not yet been updated for version 1.0.0
+When does session clear the backing store? When the session expires? After you close the browser and the session cookie is deleted - what happens to your session data?
 
-By [Steve Smith](http://ardalis.com)
+`IdleTimeout` is  independent of the cookie's expiration.
+Is there a way to set the session ID cookie timeout? What happens to the session cookie after the session is expired?
```

Comment:
Session cookies by definition have no specified timeout, the are deleted when the browser session ends.

If a cookie is received for an expired session then a new session is created using the same cookie.